ref/general/packaging-deploying.adoc

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: You can choose how to package and deploy your application with Open Liberty to meet your needs.
+:seo-title: Packaging and deploying applications
+:seo-description: You can choose how to package and deploy your application with Open Liberty to meet your needs.
+:page-layout: general-reference
+:page-type: general
+= Package and deploy applications with Open Liberty
+
+You can choose how to package and deploy your application with Open Liberty to meet your needs.
+
+== Package, containerize, and deploy using Maven and Docker
+
+One of the easiest ways to get your cloud-native application up and running is to use Maven to package it with the Open Liberty runtime and use Docker to containerize and deploy it.
```

Comment:
you might move the action out of "use" and into the more specific verbs- something like

One of the easiest ways to get your cloud-native application up and running is to package it with Maven and containerize and deploy it with Docker .

maybe add a sentence or two about why you'd want to package the app with the server and deploy in a container? What's the advantage? Does it make the application more portable, lightweight?

Also- explain Maven before you introduce the plug-in